### PR TITLE
Models for plane_reserve, hotel_reserve

### PR DESCRIPTION
**Pull request by tttthomasssss**
Adds first models (not based on MTurk data) for `hotel_reserve` and `plane_reserve`

### CODE REVIEW

File: resources/plane_reserve/intent2reply.json [hunk 1]
```diff
@@ -1,4 +1,10 @@
 {
     "hello": "Hello, how can I help?",
     "ask_name": "May I have your name, please?",
+    "plane_ask_flight_id": "Can I have your flight ID, please?",
+    "plane_flight_available": "The flight is available. Should I reserve it for you?",
+    "plane_flight_unavailable": "Sorry, but the flight with id '{flight_id:i}' is not available any more.",
+    "plane_reservation_succeeded": "Right, your reservation completed successfully!",
```
Comment (JEM-Mosig) [inline resources/plane_reserve/intent2reply.json]: This sounds a bit odd. Maybe "Your flight is now reserved." or "Your reservation was successful." Although the first is more specific, and therefore better, I think.

**Comment by JEM-Mosig** [comment]
@tttthomasssss I also have another training example for `hotel_provide_search_result`: "How about the 5-star Hilton Hotel at moderate cost in the Center?"

**Comment by JEM-Mosig** [comment]
Also, I just notice that some item descriptions (e.g. for the hotel) describe more than the properties mentioned in the schema. Could you reduce that, please? (e.g. "it does not accept reservations" should not be part of `hotel_provide_search_result`, but perhaps part of a separate intent).

**Comment by JEM-Mosig** [comment]
@tttthomasssss One more thing: The `ride_status` templates don't work, because the item returned by the API does not contain service provider nor driver name (see, e.g., `ride_provide_booking_status`). The template should be something like "{ride_status:s} ({ride_wait:s})"

**Comment by JEM-Mosig** [comment]
And a final note 😄 
I think it would be good to have a generic "Sorry, I cannot help you with that." intent in every task/domain.

**Comment by tttthomasssss** [comment]
@JEM-Mosig For the two comments before the last, are there some example KB Items for `hotel_search` and `ride_status` somewhere in the repo? I've usually used all of the information in a `db` item for the suggestions.

**Comment by JEM-Mosig** [comment]
@tttthomasssss 
> @JEM-Mosig For the two comments before the last, are there some example KB Items for `hotel_search` and `ride_status` somewhere in the repo? I've usually used all of the information in a `db` item for the suggestions.

Ah, right. Sorry, I should have explained. You find the things that end up in a KB item here: https://github.com/RasaHQ/ParlAI/blob/d1f7fc7c57d8dfc2505c9dee72d984f163ddd2e7/parlai/mturk/tasks/woz/knowledgebase/apis/hotel_search.json#L70 ( and similar files for the other tasks). 

I kind of learned along the way and didn't know about ParlAI or frontend development in the beginning. Therefore, the entire code base is really messy, and if we had the time I would completely rewrite the entire thing, but for now we have to live with it :/

**Comment by tttthomasssss** [comment]
Right, so I think I have addressed all issues you've raised, and I made a few other changes, such as finally moving the training data for all models under version control.

**Review by JEM-Mosig — APPROVED** [review]

**Pull request merged**